### ISSUE TITLE
The training set of using different pretraining datasets.

### ISSUE THREAD

**Issue by ShiYaya** [zdou0830/METER#27]
When I tried to reproduce the results in Table 17, I found that using the default learning rate and only using the coco pertaining dataset worked extremely poorly on downstream tasks.

So, I would like to ask, do you set different training parameters (eg, lr, bs, max epoch, etc) for different pre-training datasets?

**Comment by zdou0830** [comment]
Hi, I used CLIP-32 as the image encoder and BERT as the text encoder and pretrained the model for 50k steps as in the paper. I think I didn't change the other parameters but I don't recall the details.

**Comment by ShiYaya** [comment]
Thanks for your reply!
I found a more efficient setting for coco (8*V100 need 13h):
I set it up as follows and can achieve comparable performance:
warm_steps = 0.1
batch_size = 512 
max_epoch = 30
max_steps = None 

hhh!!!!

**Comment by mactavish91** [comment]
> Thanks for your reply! I found a more efficient setting for coco (8*V100 need 13h): I set it up as follows and can achieve comparable performance: warm_steps = 0.1 batch_size = 512 max_epoch = 30 max_steps = None
> 
> hhh!!!!

@ShiYaya Hello, have you conducted pre training on other data, and can you achieve the experimental results in the paper?
